python/chronos/src/bigdl/chronos/forecaster/lstm_forecaster.py

```diff
@@ -49,7 +49,8 @@ def __init__(self,
                  seed=None,
                  distributed=False,
                  workers_per_node=1,
-                 distributed_backend="ray"):
+                 remote_distributed_backend="ray",
+                 local_distributed_backend="subprocess"):
```

Comment:
We should not change the API, just
```python
self.remote_distributed_backend = distributed_backend
self.local_distributed_backend = "subprocess"
```

Reply:
please remove this `local_distributed_backend` parameter since we don't want our users to set this parameter.

Reply:
Fixed, removed `local_distributed_backend` and some doc.